packages/kube-client/lib/resources/GardenerCore.js

```diff
@@ -66,6 +66,10 @@ class Shoot extends mix(GardenerCore).with(NamespaceScoped, Readable, Observable
     return this.create([namespace, name, 'adminkubeconfig'], body, options)
   }
 
+  patchBindingResource (namespace, name, body, options) {
```

Comment:
Add a test file `resources.test.js`for this :
```js
//
// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
//
// SPDX-License-Identifier: Apache-2.0
//

'use strict'

const { mockClient } = require('@gardener-dashboard/request')
const { Shoot } = require('../lib/resources/GardenerCore')

describe('kube-client', () => {
  describe('resources', () => {
    describe('core.gardener.cloud', () => {
      describe('shoots', () => {
        const url = 'http://example.org'
        const namespace = 'default'
        const name = 'test'
        const data = { foo: 'bar' }

        let shoot

        beforeEach(() => {
          shoot = new Shoot({ url })
          mockClient.request.mockImplementation((...args) => args)
        })

        it('should patch binding subresource', async () => {
          const args = await shoot.patchBindingResource(namespace, name, data)
          expect(args).toEqual([
            `namespaces/${namespace}/shoots/${name}/binding`,
            {
              method: 'patch',
              headers: {
                'content-type': 'application/merge-patch+json'
              },
              json: data
            }
          ])
        })

        it('should create an adminkubeconfig subresource', async () => {
          const args = await shoot.createAdminKubeconfigRequest(namespace, name, data)
          expect(args).toEqual([
            `namespaces/${namespace}/shoots/${name}/adminkubeconfig`,
            {
              method: 'post',
              json: data
            }
          ])
        })
      })
    })
  })
})

```

Reply:
as we removed `patchBindingResource` function and `createAdminKubeconfigRequest` is tested indirectly, I think we do not need to add this.

Reply:
added file, removed test for binding subresource